src/content/changes/highlights/2022-07-15

```diff
@@ -0,0 +1,19 @@
+---
+date: 2022-07-15T10:00:00
+title: Highlights for the week ending July 15, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+
+## Apps
+
```

Comment:
```suggestion
- [starboard-exporter](https://github.com/giantswarm/starboard-exporter) version [v0.5.1](https://github.com/giantswarm/starboard-exporter/blob/main/CHANGELOG.md#051---2022-07-13) allows users to control which resource types are watched by the exporter, so that it can be deployed with either `trivy-operator` or `starboard`.

```